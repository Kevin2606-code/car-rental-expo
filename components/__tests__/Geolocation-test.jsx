import * as React from "react";
import renderer from "react-test-renderer";

import Geolocation from "../Geolocation";

it(`Geolocation renders correctly`, () => {
  const tree = renderer.create(<Geolocation />).toJSON();

  expect(tree).toMatchSnapshot();
});
