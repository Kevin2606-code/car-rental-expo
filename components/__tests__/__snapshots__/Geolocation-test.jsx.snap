// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CarList renders correctly 1`] = `
<View
  accessibilityState={
    {
      "busy": undefined,
      "checked": undefined,
      "disabled": undefined,
      "expanded": undefined,
      "selected": undefined,
    }
  }
  accessibilityValue={
    {
      "max": undefined,
      "min": undefined,
      "now": undefined,
      "text": undefined,
    }
  }
  accessible={true}
  collapsable={false}
  focusable={true}
  onBlur={[Function]}
  onClick={[Function]}
  onFocus={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    {
      "backgroundColor": "#ffffff",
      "borderColor": "#999999",
      "borderRadius": 6,
      "borderWidth": 1,
      "elevation": 5,
      "margin": 10,
      "paddingHorizontal": 10,
      "paddingVertical": 30,
    }
  }
>
  <View
    style={
      {
        "alignItems": "center",
        "flexDirection": "row",
        "gap": 20,
        "justifyContent": "flex-start",
      }
    }
  >
    <View>
      <Image
        source={1}
        style={
          {
            "height": 60,
            "objectFit": "contain",
            "width": 100,
          }
        }
      />
    </View>
    <View>
      <Text
        style={
          {
            "fontSize": 16,
          }
        }
      >
        Toyota Avanza
      </Text>
      <View
        style={
          {
            "alignItems": "baseline",
            "flexDirection": "row",
            "gap": 10,
            "justifyContent": "flex-start",
          }
        }
      >
        <View
          style={
            {
              "alignItems": "center",
              "flexDirection": "row",
              "gap": 2,
            }
          }
        >
          <Text />
          <Text
            style={
              {
                "color": "#8A8A8A",
                "fontFamily": "PoppinsBold",
                "fontSize": 12,
                "paddingLeft": 5,
                "paddingTop": 5,
              }
            }
          >
            4
          </Text>
        </View>
        <View
          style={
            {
              "alignItems": "center",
              "flexDirection": "row",
              "gap": 2,
            }
          }
        >
          <Text />
          <Text
            style={
              {
                "color": "#8A8A8A",
                "fontFamily": "PoppinsBold",
                "fontSize": 12,
                "paddingLeft": 5,
                "paddingTop": 5,
              }
            }
          >
            2
          </Text>
        </View>
      </View>
      <Text
        style={
          {
            "color": "#5cb85f",
            "fontFamily": "PoppinsBold",
            "fontSize": 16,
            "paddingTop": 10,
          }
        }
      >
        RpÂ 100.000,00
      </Text>
    </View>
  </View>
</View>
`;

exports[`Geolocation renders correctly 1`] = `
<Text
  style={
    {
      "color": "#ffffff",
      "fontFamily": "PoppinsBold",
      "fontSize": 14,
    }
  }
>
  Location Loading.....
</Text>
`;
